### PR TITLE
Added a section about connectors stackdriver logging configuration

### PR DESCRIPTION
**Pull request by jonathanlukas**
## Description

There is an [open PR](https://github.com/camunda/connectors/pull/1298) to enable stackdriver logging on connectors.

This is the documentation for this.

## When should this change go live?

- [x] This change is not yet live and should not be merged until the PR is merged https://github.com/camunda/connectors/pull/1298 (apply `hold` label or convert to draft PR)?
- [ ] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [x] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Review by christinaausley — APPROVED** [review]

**Comment by christinaausley** [comment]
No issues with this, just let me know when you are ready to merge 👍

File: docs/self-managed/connectors-deployment/connectors-configuration.md [hunk 1]
```diff
@@ -243,3 +243,9 @@ configuration required from the user.
 
 If you want to restrict the Connector Runtime Inbound Connector feature to a single tenant or multiple tenants
 you have to use Identity and assign the tenants the Connector application should have access to.
+
+## Logging
+
+### Google Stackdriver (JSON) logging
+
+To enable Google Stackdriver compatible JSON logging, set the environment variable `CONNECTORS_LOG_APPENDER=Stackdriver` on the Connector Runtime.
```
Comment (sbuettner) [inline docs/self-managed/connectors-deployment/connectors-configuration.md:251]:
```suggestion
To enable Google Stackdriver compatible JSON logging, set the environment variable `CONNECTORS_LOG_APPENDER=stackdriver` on the Connector Runtime.
```

File: versioned_docs/version-8.3/self-managed/connectors-deployment/connectors-configuration.md [hunk 2]
```diff
@@ -243,3 +243,9 @@ configuration required from the user.
 
 If you want to restrict the Connector Runtime Inbound Connector feature to a single tenant or multiple tenants
 you have to use Identity and assign the tenants the Connector application should have access to.
+
+## Logging
+
+### Google Stackdriver (JSON) logging
+
+To enable Google Stackdriver compatible JSON logging, set the environment variable `CONNECTORS_LOG_APPENDER=Stackdriver` on the Connector Runtime.
```
Comment (sbuettner) [inline versioned_docs/version-8.3/self-managed/connectors-deployment/connectors-configuration.md:251]:
```suggestion
To enable Google Stackdriver compatible JSON logging, set the environment variable `CONNECTORS_LOG_APPENDER=stackdriver` on the Connector Runtime.
```

**Review by sbuettner — APPROVED** [review]

**Comment by akeller** [comment]
The referenced PR is merged. Is this good to merge too?

**Pull request merged**